quickstart/java/android/keyword-recognizer/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart: Keyword Recognition in Java on Android
+
+This sample demonstrates how to recognize keywords with Java using the Speech SDK for Android.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-java-android) on the SDK documentation page which describes how to build this sample from scratch in Android Studio.
+
+> **Note:**
+> this sample is *not* for the Speech Devices SDK and the Roobo device.
+> If you are looking for further information on these, visit the [Speech Devices SDK](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-devices-sdk) page.
+
+## Prerequisites
+
+* A PC (Windows, Linux, Mac) capable to run Android Studio.
+* Version 3.1 of [Android Studio](https://developer.android.com/studio/).
+* An ARM-based Android device (API 23: Android 6.0 Marshmallow or higher) [enabled for development](https://developer.android.com/studio/debug/dev-options) with a working microphone.
```

Comment:
>ARM-based [](start = 5, length = 9)

could be also x86 emulator

Reply:
We now have x86, ARM and ARM64 so it should work on pretty much every phone/emulator. Will remove the architecture from that statement